public/src/reducers/calls.jsx

```diff
@@ -41,17 +66,27 @@ export function volunteerCallsReducer(state = defaultCalls, action) {
     case CLEAR_CALL_CURRENT:
       return {
         ...state,
-        current_call: undefined
+        ...defaultCalls
       };
-    case CLEAR_NEXT_CALL:
+    case CLEAR_COUNT_CALLS:
       return {
         ...state,
-        next_call: undefined
+        calls_made: payload
       };
-    case CLEAR_COUNT_CALLS:
+    case UPDATE_CALL_STATUS:
       return {
         ...state,
-        calls_made: payload
+        status: payload
+      };
+    case UPDATE_CALL_OUTCOME:
+      return {
+        ...state,
+        outcome: payload
+      };
+    case SET_CALL_CONTACT_INFO:
+      return {
+        ...state,
+        contact_name: payload.name
```

Comment:
Would suggest storing this as current_contact.name or current_contact_name or something like that to avoid namespace pollution.

Reply:
done